### ISSUE TITLE
Less accurate boxes with better detection

### ISSUE THREAD

**Issue by AlasterMeehan** [AlexeyAB/darknet#6315]
"to make the detected bounded boxes more accurate, you can add 3 parameters ignore_thresh = .9 iou_normalizer=0.5 iou_loss=giou to each [yolo] layer and train, it will increase mAP@0.9, but decrease mAP@0.5."

The above states that to improve accuracy to adjust the 3 parameters above.

If box accuracy is not important and you are only interested in detection is it possible to get better detection with less accuracy, ie improve mAP@0.3 or mAP@0.1?
Should other parameters be adjusted or just "ignore_thresh"?
What is recomended?

**Comment by scamianbas** [comment]
Yes, what is recommended if you want to have bounding boxes edges less accurate but increase bounding boxes content classification ?
Thnak you very much.

**Comment by AlexeyAB** [comment]
Set
[yolo]
ignore_thresh =0.5
iou_normalizer=0.02
iou_loss=mse

**Comment by cpsu00** [comment]
@AlexeyAB Do this setting works for yolov4-csp?
